src/clarity-icons/clarity-icons.spec.ts

```diff
@@ -373,15 +356,134 @@ describe("ClarityIcons", () => {
             expect(clarityIcon.style.width).toBe("25px");
             expect(clarityIcon.style.height).toBe("25px");
         });
+
+        it("should be able to use non-svg template", () => {
+            const testShape = `<img src="arrow-icon.png" alt="arrow-icon" width="42" height="42">`;
+            ClarityIcons.add({"test-shape": testShape});
+
+            const clarityIcon = document.createElement("clr-icon");
+            clarityIcon.setAttribute("shape", "test-shape");
+
+            expect(removeWhitespace(clarityIcon.innerHTML)).toBe(removeWhitespace(testShape));
+        });
+
+        it("should not change non-svg template if title attribute is specified", () => {
+            const testShape = `<img src="arrow-icon.png" alt="arrow-icon" width="42" height="42">`;
+            ClarityIcons.add({"test-shape": testShape});
+
+            const clarityIcon = document.createElement("clr-icon");
+            clarityIcon.setAttribute("shape", "test-shape");
+            clarityIcon.setAttribute("title", "my-custom-title");
+
+            expect(removeWhitespace(clarityIcon.innerHTML)).toBe(removeWhitespace(testShape));
+        });
+
+        it("should not inject anything if the custom title is not given", () => {
+            const customTitle = "my-custom-title";
```

Comment:
You're not using `customTitle` here. Happens to all of us with test copy-paste. 😛

Reply:
fixed that.